cartographer/mapping/map_builder.cc

```diff
@@ -181,9 +181,7 @@ void MapBuilder::SerializeState(io::ProtoStreamWriter* const writer) {
         // TODO(whess): Handle trimmed data.
         range_data_proto->mutable_node_id()->set_trajectory_id(trajectory_id);
         range_data_proto->mutable_node_id()->set_node_index(node_index);
-        const auto& data = *node_data[trajectory_id][node_index].constant_data;
-        *range_data_proto->mutable_range_data() =
-            sensor::ToProto(data.range_data);
+        range_data_proto->mutable_range_data();
```

Comment:
I don't understand. Is this not a no-op?

Reply:
We don't have any range data anymore and I would like to send a follow up PR for serializing the new data to keep this PR small and focused. This is adding the proto message without adding any data in it. So strictly speaking not a no-op.